client/header/activity-panel/panels/orders.js

```diff
@@ -162,29 +179,30 @@ function OrdersPanel( { orders, isRequesting, isError } ) {
 }
 
 OrdersPanel.propTypes = {
-	orders: PropTypes.instanceOf( Map ).isRequired,
+	orders: PropTypes.array.isRequired,
 	isError: PropTypes.bool,
 	isRequesting: PropTypes.bool,
 };
 
 OrdersPanel.defaultProps = {
-	orders: new Map(),
+	orders: [],
 	isError: false,
 	isRequesting: false,
 };
 
 export default compose(
 	withSelect( select => {
-		const { getItems, getItemsError, isGetItemsRequesting } = select( 'wc-api' );
+		const { getReportItems, getReportItemsError, isReportItemsRequesting } = select( 'wc-api' );
 		const ordersQuery = {
 			page: 1,
 			per_page: QUERY_DEFAULTS.pageSize,
-			status: 'processing',
+			status_is: 'processing',
```

Comment:
Should this only grab orders that are `processing`?  What about `on-hold` items?
/cc @timmyc @jeffstieler

Reply:
I see there's some discussion around this in https://github.com/woocommerce/wc-admin/issues/935.  An actionable list of orders makes sense to me; I believe `on-hold` may fall into this realm since it often requires manually checking with your bank and confirming payment.

Does that sound accurate?  Any other "actionable" statuses?

Reply:
+1 for having `on-hold` being considered "actionable", but ideally thinking we should either:

- Expose this as a configurable option on the settings page
- Or make it a fitlerable array of statuses that gets passed in via `wcSettings` and consumed in the component(s)

Reply:
I was also thinking that maybe this should be a setting option for users, so I'm completely on board with that option.  Issue opened around that [here](https://github.com/woocommerce/wc-admin/issues/1593).